### ISSUE TITLE
Does the symbol '⊕' in your paper means Blinear Upsample + ele-wise sum?

### ISSUE THREAD

**Issue by wudejian789** [qijiezhao/M2Det#93]
Thank you very much for your work. I found some questions. And if you have any time for replying, that's great. 
I saw the symbol '⊕' in your paper means **Blinear Upsample + ele-wise sum**, while in your code: M2Det/layers/nn_utils.py line 71 this is **Nearest Upsample + ele-wise**. So I want to know that which one is the best one and dose there any influences for the two usage? 
In addition, I found a possible error in your paper. In the formula 1 of your paper, should the function **F** be **FFMv2**?
Thank you again for your contribution. Good luck!

**Comment by shuang0112** [comment]
I found the same question. Have you changed the code to run the project?

**Comment by wudejian789** [comment]
> I found the same question. Have you changed the code to run the project?

No. I'm reproducting M2Det reference the paper and the github. The differences between paper and github is because the author has changed the model architecture to achieve a better performances. But according the author, the new architecture is not good as before. Specifically, you can refer to the previous issues. Someone mentioned similar issues and the author replied it.

**Comment by shuang0112** [comment]
Thanks for your reply！And I am also wondering if you found the structure of TUM in the code is different from paper? maybe I make some mistakes, so I want to have a discussion with you，thanks!

**Comment by wudejian789** [comment]
> Thanks for your reply！And I am also wondering if you found the structure of TUM in the code is different from paper? maybe I make some mistakes, so I want to have a discussion with you，thanks!

Yes, I found it. The structure of TUM in github is different from the paper.  You can have a look at issue #36 . The author said "There indeed exists difference, the arch of original paper is a little better while the arch in this repo is more fancy I think. You can try to modify it in the code."

**Comment by shuang0112** [comment]
Okay , thank you very much!
